src/msg/async/dpdk/DPDK.cc

```diff
@@ -529,31 +532,21 @@ bool DPDKQueuePair::init_rx_mbuf_pool()
     // 2) Bind data buffers to each of them.
     // 3) Return them back to the pool.
     //
-    for (int i = 0; i < mbufs_per_queue_rx; i++) {
-      rte_mbuf* m = rte_pktmbuf_alloc(_pktmbuf_pool_rx);
-      assert(m);
-      rte_mbuf_refcnt_set(m, 1);
-      _rx_free_bufs.push_back(m);
+    int ret = snprintf(mz_name, sizeof(mz_name),
```

Comment:
why not just use `"rx_buffer"`? IMO, no need to print a string here. also, please note, "data" is the plural of "datum".

Reply:
here, allocate memory just for data part, not include rx_buffer's header.  so use the rx_buffer_data as the name.  correct "datas" as " data".